service/mcs/persistent/src/main/resources/create_cassandra_schema.cql

```diff
@@ -41,4 +41,11 @@ CREATE TABLE representation_versions (
 	creation_date timestamp,
 	PRIMARY KEY (cloud_id, schema_id, version_id)
 );
-CREATE INDEX representations_provider_id ON representation_versions (provider_id);
\ No newline at end of file
+CREATE INDEX representations_provider_id ON representation_versions (provider_id);
+
+CREATE TABLE data_sets_representations_names(

```

Comment:
First of all having a document that we agree upon it and then implementing it differently, for me doesn't make any sense..
For this table, where is the creation_date and updated_timestamp that are related to the representation? 
And where is the representation_id on the primary key? If I want to check if a representation name is in a specific dataset I want to be able to have a fast direct call, not retrieve a set of 100 representation names and search if it's inside the set.